src/lib/image-uri.ts

```diff
@@ -1,17 +1,17 @@
 export const IPFS_GATEWAY = 'ipfs.io';
 
-export function formatImgUri(origin: string) {
-  if (origin.startsWith('ipfs://')) {
+export function formatImgUri(origin?: string) {
+  if (origin && origin.startsWith('ipfs://')) {
     return `https://${IPFS_GATEWAY}/ipfs/${origin.substring(7)}/`;
   }
-  return origin;
+  return origin ?? '';
 }
 
-export function sanitizeImgUri(url: string, x = 64, y = 64) {
+export function sanitizeImgUri(url: string | undefined, x = 64, y = 64): string {
```

Comment:
in one place you are doing `prop?: string`, in another `prop: string | undefined`.  
Make it one way and set a default value to omit redundant checks and fallbacks